indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -438,30 +453,85 @@ function EditableListDisplay({
       />
     );
   };
+  const renderTags = (__, editable) => {
+    const tags = editable.tags.sort((a, b) => a.code.localeCompare(b.code));
+    const labelTags = () => {
+      return (
+        <div style={{display: 'flex', flexWrap: 'nowrap'}}>
+          {tags.map(t => {
+            return (
+              <Label key={t.code} size="small" color={t.color}>
+                {t.code}
+              </Label>
+            );
+          })}
+        </div>
+      );
+    };
+
+    const colorTags = () => {
```

Comment:
Why are those functions and not just assignments? You call them right below, so there's not much benefit in making them functions...